tests/cmds/test_profile.py

```diff
@@ -530,3 +509,75 @@ def test_use_profile(runner, mock_cliprofile_namespace, profile):
         profile.name
     )
     assert f"{profile.name} has been set as the default profile." in result.output
+
+
+def test_totp_option_passes_token_to_sdk_on_profile_cmds_that_init_sdk(
+    runner, mocker, mock_cliprofile_namespace, cli_state
+):
+    totp1 = "123456"
+    totp2 = "234567"
+    mock_create_sdk = mocker.patch("code42cli.cmds.profile.create_sdk")
+    runner.invoke(
+        cli,
+        [
+            "profile",
+            "create",
+            "-n",
+            "foo",
+            "-s",
+            "bar",
+            "-u",
+            "baz",
+            "--password",
+            "testpass",
+            "--totp",
+            totp1,
```

Comment:
I am surprised this test passes. Does the `totp` validation logic not happen in the test runner?

Reply:
why would it fail?

Reply:
Because it uses the TOTP type which verifies that the value is an int with 6 digits?
Is that part mocked out or something?

Reply:
totp1 is an int (convertable from string) with 6 digits: https://github.com/code42/code42cli/pull/310/files/e15cb0006fb93dd1dd7e65d50f542f4bdeb03f78#diff-da10b77cdc04e2d1e4cee69562deba89b7e1eb31322da3129157212e08f3d2fcR517

Reply:
Agh so sorry, I thought that was `"totp1"`